lib/gui/index.html

```diff
@@ -54,10 +54,11 @@
     <div class="section-loader"
       ng-controller="StateController as state"
       ng-class="{
-        isFinish: state.is('success')
+        isFinish: state.currentName === 'success'
       }">
       <safe-webview
-        src="'https://etcher.io/success-banner/'"></safe-webview>
+        src="'https://etcher.io/success-banner/'"
+        refresh-now="state.previousName === 'success'"></safe-webview>
```

Comment:
I'm going to test this later, but what happens if this always evaluates to `true`? Will the webview keep refreshing?